pkg/libartifact/store/store.go

```diff
@@ -0,0 +1,409 @@
+//go:build !remote
+
+package store
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/containers/common/libimage"
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/image/v5/oci/layout"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/podman/v5/pkg/libartifact"
+	libartTypes "github.com/containers/podman/v5/pkg/libartifact/types"
+	"github.com/containers/storage"
+	"github.com/go-openapi/runtime"
+	"github.com/opencontainers/go-digest"
+	"github.com/opencontainers/image-spec/specs-go"
+	specV1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// indexName is the name of the JSON file in root of the artifact store
+	// that describes the store's contents
+	indexName   = "index.json"
+	emptyStanza = []byte("{}")
```

Comment:
This is the same as `specV1.DescriptorEmptyJSON.Data`.

Reply:
yeah but i need a []byte representation so I was saving yet another JSON marshal.  still worth it?

Reply:
That `Data` field is already a JSON object with no fields in it, so you'd be good there.